src/SarifCli/Rules/DoNotUseFriendlyNameAsRuleId.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif.Cli.Rules
+{
+    public class DoNotUseFriendlyNameAsRuleId : SarifValidationSkimmerBase
+    {
+        public override string FullDescription => RuleResources.SV0001_DoNotUseFriendlyNameAsRuleIdDescription;
+
+        /// <summary>
+        /// SV0001
```

Comment:
I did consider SARIF, but I wasn't aware of a precedent for a 5-character prefix. Sure, I'll change it.

---
In reply to: [77239380](https://github.com/Microsoft/sarif-sdk/pull/501#discussion_r77239380) [](ancestors = 77239380)